docs/source/reference/leaderboard_faq.rst

```diff
@@ -145,3 +146,26 @@ A few things you can explore to improve the speed of transcoding include:
 A few things you can explore to improve the speed of data upload/download:
 
 - Review the :doc:`bandwidth_requirements` and consider upgrading your bandwidth 
+
+How can I get historical test results for an orchestrator using the Leaderboard API ? 
+*************************************************************************************
+
+- To get aggregated stats for an orchestrator since a particular timestamp use:
+
+::
+
+    curl https://leaderboard-serverless-git-nv-api-rework.livepeerorg.vercel.app/api/aggregated_stats?orchestrator=<ORCHESTRATOR>?since=<TIMESTAMP>?region=<REGION>
+
+- <ORCHESTRATOR> the orchestrator's Ethereum address to get the metrics for.
+- <TIMESTAMP> the timestamp at which to evaluate the query. This defaults to the last 24 hours, to get the aggregated stats for all time you can use `since=1`.
+- <REGION> the region to get results for, returns all regions by default.
+
+- To get each individual test entry for an rochestrator since a particular timestmap use:
```

Comment:
rorchestrator typo

Reply:
fixed in 74b5828d64570b09d599a456930e675c8b3a1026